src/main/java/nl/tudelft/scrumbledore/Collision.java

```diff
@@ -6,6 +6,7 @@
  * @author Jesse Tilro
  *
  */
+@SuppressWarnings("PMD.TooManyMethods")
```

Comment:
Maybe we should eventually listen to this warning by indeed refactoring and spreading the methods over different classes?